File: Ban.cpp
#include <bits/stdc++.h> // Mengimpor semua library standar

using namespace std;

class Ban{
    private:
        string merk, ukuran;
        int tahun_pembuatan;
    
    public:
        Ban(){
        }

        Ban(string merk, string ukuran, int tahun_pembuatan){
            this->merk = merk;
            this->ukuran = ukuran;
            this->tahun_pembuatan = tahun_pembuatan;
        }

        void set_merk(string merk){
            this->merk = merk;
        }

        void set_ukuran(string ukuran){
            this->ukuran = ukuran;
        }

        void set_tahun_pembuatan(int tahun_pembuatan){
            this->tahun_pembuatan = tahun_pembuatan;
        }

        string get_merk(){
            return this->merk;
        }

        string get_ukuran(){
            return this->ukuran;
        }

        int get_tahun_pembuatan(){
            return this->tahun_pembuatan;
        }

        ~Ban(){
        }
};

File: Javaaa/program/Ban.java
class Ban {
    private String merk;
    private String ukuran;
    private int tahunPembuatan;

    public Ban() {
    }

    public Ban(String merk, String ukuran, int tahunPembuatan) {
        this.merk = merk;
        this.ukuran = ukuran;
        this.tahunPembuatan = tahunPembuatan;
    }

    public void setMerk(String merk) {
        this.merk = merk;
    }

    public void setUkuran(String ukuran) {
        this.ukuran = ukuran;
    }

    public void setTahunPembuatan(int tahunPembuatan) {
        this.tahunPembuatan = tahunPembuatan;
    }

    public String getMerk() {
        return this.merk;
    }

    public String getUkuran() {
        return this.ukuran;
    }

    public int getTahunPembuatan() {
        return this.tahunPembuatan;
    }
}